diff -durN freeswan-1.4.orig/utils/setup freeswan-1.4/utils/setup
--- freeswan-1.4.orig/utils/setup	Mon May 22 06:37:57 2000
+++ freeswan-1.4/utils/setup	Sat Jun 10 13:07:02 2000
@@ -37,6 +37,8 @@
 	export PATH
 fi
 
+. /etc/rc.d/init.d/functions
+
 me='ipsec setup'		# for messages
 
 # make sure output of (e.g.) ifconfig is in English
@@ -368,7 +370,7 @@
 # logging control
 logit() {
 	IPSECsyslog=${IPSECsyslog-daemon.error}
-	logger -s -p $IPSECsyslog -t ipsec_setup 2>&1
+	logger -p $IPSECsyslog -t ipsec_setup 2>&1
 }
 logonly() {
 	IPSECsyslog=${IPSECsyslog-daemon.error}
@@ -391,12 +393,14 @@
 	# Start things rolling.
 	# (Warning, changes to this log message may affect barf.)
 	version="`ipsec --version | awk 'NR == 1 { print $3 }'`"
+ 	msg_starting "FreeS/WAN IPSEC"; busy
 	echo "Starting FreeS/WAN IPSEC $version..." | logit
 	rm -f $info
 	if test ! -r /dev/random
 	then
 		echo "...unable to start FreeS/WAN IPSEC, no /dev/random!" |
 									logit
+ 		fail
 		exit 1
 	fi
 	startklips 2>&1 | logit
@@ -434,10 +438,12 @@
 	fore)	plutogo 2>&1 | logit	;;
 	esac
 	echo "...FreeS/WAN IPSEC started" | logonly
+ 	ok
 	;;
 
   stop|--stop)
 	# Shut things down.
+ 	msg_stopping "FreeS/WAN IPSEC" ; busy
 	echo "Stopping FreeS/WAN IPSEC..." | logit
 	if test " $IPSECforwardcontrol" = " yes"
 	then
@@ -485,8 +491,14 @@
 	fi
 	rm -f $info
 	echo "...FreeS/WAN IPSEC stopped" | logonly
+ 	ok
 	;;
 
+  status|--status)
+ 	status pluto
+ 	ipsec look
+ 	;;
+ 	
   restart|--restart)
 	$0 stop
 	$0 start
