diff -Naur freeswan-1.95/Makefile.inc freeswan-1.95-p/Makefile.inc
--- freeswan-1.95/Makefile.inc	Fri Feb  1 04:26:40 2002
+++ freeswan-1.95-p/Makefile.inc	Thu Feb  7 19:23:59 2002
@@ -45,7 +45,7 @@
 DESTDIR=
 
 # "local" part of tree, used in building other pathnames
-INC_USRLOCAL=/usr/local
+INC_USRLOCAL=/usr
 
 # PUBDIR is where the "ipsec" command goes; beware, many things define PATH
 # settings which are assumed to include it (or at least, to include *some*
@@ -80,10 +80,7 @@
 # will finally be (so utils/Makefile can create a symlink in BINDIR to the
 # place where the boot/shutdown script will finally be, rather than the
 # place where it is installed).
-FINALRCDIR=$(shell for d in $(INC_RCDIRS) ; \
-		do if test -d $(DESTDIR)/$$d ; \
-		then echo $$d ; exit 0 ; \
-		fi ; done ; echo $(INC_RCDEFAULT) )
+FINALRCDIR=/etc/rc.d/init.d
 RCDIR=$(DESTDIR)$(FINALRCDIR)
 
 
@@ -124,8 +121,8 @@
 
 # extra compile flags, for userland and kernel stuff, e.g. -g for debug info
 # (caution, this stuff is still being sorted out, will change in future)
-USERCOMPILE=-g -O3
-KLIPSCOMPILE=-O3
+USERCOMPILE=$(OPT_FLAGS)
+KLIPSCOMPILE=$(OPT_FLAGS)
 
 # extra options for use in kernel build
 KERNMAKEOPTS=
diff -Naur freeswan-1.95/pluto/Makefile freeswan-1.95-p/pluto/Makefile
--- freeswan-1.95/pluto/Makefile	Tue Nov 27 17:33:42 2001
+++ freeswan-1.95-p/pluto/Makefile	Thu Feb  7 19:26:29 2002
@@ -71,7 +71,8 @@
 
 CPPFLAGS = $(HDRDIRS) $(BYTE_ORDER) \
 	-DPLUTO -DKLIPS -DDODGE_DH_MISSING_ZERO_BUG \
-	-DDEBUG -DGCC_LINT # -DLEAK_DETECTIVE
+	-DDEBUG -DGCC_LINT -DSHARED_SECRETS_FILE='"/etc/freeswan/ipsec.secrets"'
+	# -DLEAK_DETECTIVE
 
 ALLFLAGS = $(CPPFLAGS) $(CFLAGS)
 
