diff -Naur freeswan-1.96.orig/Makefile.inc freeswan-1.96/Makefile.inc
--- freeswan-1.96.orig/Makefile.inc	Mon Mar 18 09:32:50 2002
+++ freeswan-1.96/Makefile.inc	Mon Mar 18 09:40:32 2002
@@ -45,7 +45,7 @@
 DESTDIR=
 
 # "local" part of tree, used in building other pathnames
-INC_USRLOCAL=/usr/local
+INC_USRLOCAL=/usr
 
 # PUBDIR is where the "ipsec" command goes; beware, many things define PATH
 # settings which are assumed to include it (or at least, to include *some*
@@ -61,12 +61,12 @@
 # location within INC_USRLOCAL
 INC_MANDIR=man
 # the full pathname
-MANTREE=$(DESTDIR)$(INC_USRLOCAL)/$(INC_MANDIR)
+MANTREE=$(DESTDIR)$(INC_USRLOCAL)/share/$(INC_MANDIR)
 # all relevant subdirectories of MANTREE
 MANPLACES=man3 man5 man8
 
 # where configuration files go
-FINALCONFDIR=/etc
+FINALCONFDIR=/etc/ipsec
 CONFDIR=$(DESTDIR)$(FINALCONFDIR)
 
 # An attempt is made to automatically figure out where boot/shutdown scripts 
@@ -74,17 +74,14 @@
 # If none of those exists (or INC_RCDIRS is empty), INC_RCDEFAULT gets them.
 # With a non-null DESTDIR, INC_RCDEFAULT will be used unless one of the
 # INC_RCDIRS directories has been pre-created under DESTDIR.
-INC_RCDIRS=/etc/rc.d/init.d /etc/rc.d /etc/init.d /sbin/init.d
-INC_RCDEFAULT=/etc/rc.d/init.d
+#INC_RCDIRS=/etc/rc.d/init.d /etc/rc.d /etc/init.d /sbin/init.d
+#INC_RCDEFAULT=/etc/rc.d/init.d
 
 # RCDIR is where boot/shutdown scripts go; FINALRCDIR is where they think
 # will finally be (so utils/Makefile can create a symlink in BINDIR to the
 # place where the boot/shutdown script will finally be, rather than the
 # place where it is installed).
-FINALRCDIR=$(shell for d in $(INC_RCDIRS) ; \
-		do if test -d $(DESTDIR)/$$d ; \
-		then echo $$d ; exit 0 ; \
-		fi ; done ; echo $(INC_RCDEFAULT) )
+FINALRCDIR=/etc/rc.d/init.d
 RCDIR=$(DESTDIR)$(FINALRCDIR)
 
 
@@ -125,8 +122,8 @@
 
 # extra compile flags, for userland and kernel stuff, e.g. -g for debug info
 # (caution, this stuff is still being sorted out, will change in future)
-USERCOMPILE=-g -O3
-KLIPSCOMPILE=-O3
+USERCOMPILE=$(OPT_FLAGS)
+KLIPSCOMPILE=$(OPT_FLAGS)
 
 # extra options for use in kernel build
 KERNMAKEOPTS=
