--- freeswan-1.97/utils/_confread	Tue Mar 26 18:49:42 2002
+++ freeswan-1.97/utils/_confread.org	Wed May  1 18:30:05 2002
@@ -14,7 +14,7 @@
 #
 # RCSID $Id$
 #
-# Extract configuration info from /etc/ipsec.conf, repackage as assignments
+# Extract configuration info from /etc/ipsec/ipsec.conf, repackage as assignments
 # to shell variables or tab-delimited fields.  Success or failure is reported
 # inline, as extra data, due to the vagaries of shell backquote handling.
 # In the absence of --varprefix, output is tab-separated fields, like:
@@ -41,7 +41,7 @@
 #	exit 1
 # fi
 
-config=${IPSEC_CONFS-/etc}/ipsec.conf
+config=/etc/ipsec/ipsec.conf
 include=yes
 type=conn
 fieldfmt=yes
--- freeswan-1.97/utils/auto	Tue Mar 26 06:07:48 2002
+++ freeswan-1.97/utils/auto.org	Wed May  1 18:30:51 2002
@@ -24,7 +24,7 @@
 	other options: [--config ipsecconfigfile] [--verbose] [--show]"
 
 showonly=
-config=
+config=/etc/ipsec/ipsec.conf
 info=/var/run/ipsec.info
 shopts=
 noinclude=
--- freeswan-1.97/utils/barf	Tue Mar 26 18:49:42 2002
+++ freeswan-1.97/utils/barf.org	Wed May  1 18:32:11 2002
@@ -16,7 +16,7 @@
 
 KERNSRC=${KERNSRC-/usr/src/linux}
 LOGS=${LOGS-/var/log}
-CONFS=${IPSEC_CONFS-/etc}
+CONFS=/etc/ipsec/ipsec.conf
 me="ipsec barf"
 
 # message patterns that start relevant parts of logs
--- freeswan-1.97/utils/showhostkey	Fri Oct 26 18:17:02 2001
+++ freeswan-1.97/utils/showhostkey.org	Wed May  1 18:33:00 2002
@@ -17,7 +17,7 @@
 me="ipsec showhostkey"
 usage="Usage: $me [--file secrets] [--left] [--right] [--txt gateway] [--id id]"
 
-file=/etc/ipsec.secrets
+file=/etc/ipsec/ipsec.secrets
 fmt="dns"
 gw=
 id=
