diff -durN freeswan-1.3.orig/utils/_confread freeswan-1.3/utils/_confread
--- freeswan-1.3.orig/utils/_confread	Tue Feb  8 22:22:30 2000
+++ freeswan-1.3/utils/_confread	Tue Apr 25 11:59:19 2000
@@ -41,7 +41,7 @@
 #	exit 1
 # fi
 
-config=/etc/ipsec.conf
+config=/etc/freeswan/ipsec.conf
 include=yes
 type=conn
 fieldfmt=yes
diff -durN freeswan-1.3.orig/utils/auto freeswan-1.3/utils/auto
--- freeswan-1.3.orig/utils/auto	Tue Feb  8 22:22:30 2000
+++ freeswan-1.3/utils/auto	Tue Apr 25 11:59:59 2000
@@ -24,7 +24,7 @@
 	other options: [--config ipsecconfigfile] [--verbose] [--show]"
 
 showonly=
-config=/etc/ipsec.conf
+config=/etc/freeswan/ipsec.conf
 info=/var/run/ipsec.info
 shopts=
 noinclude=
diff -durN freeswan-1.3.orig/utils/barf freeswan-1.3/utils/barf
--- freeswan-1.3.orig/utils/barf	Tue Feb  8 22:22:30 2000
+++ freeswan-1.3/utils/barf	Tue Apr 25 12:00:35 2000
@@ -79,9 +79,9 @@
 _________________________
 ipsec showdefaults
 _________________________
-ipsec _include /etc/ipsec.conf | ipsec _keycensor
+ipsec _include /etc/freeswan/ipsec.conf | ipsec _keycensor
 _________________________
-ipsec _include /etc/ipsec.secrets | ipsec _secretcensor
+ipsec _include /etc/freeswan/ipsec.secrets | ipsec _secretcensor
 _________________________
 cat /proc/net/dev
 _________________________
diff -durN freeswan-1.3.orig/utils/manual freeswan-1.3/utils/manual
--- freeswan-1.3.orig/utils/manual	Tue Feb  8 22:22:31 2000
+++ freeswan-1.3/utils/manual	Tue Apr 25 12:01:37 2000
@@ -28,7 +28,7 @@
 export LANG LC_ALL
 
 showonly=
-config=/etc/ipsec.conf
+config=/etc/freeswan/ipsec.conf
 info=/var/run/ipsec.info
 shopts=
 other=0
