diff -Naur freeswan-1.95/utils/_confread freeswan-1.95-p/utils/_confread
--- freeswan-1.95/utils/_confread	Sun Oct 14 01:42:50 2001
+++ freeswan-1.95-p/utils/_confread	Thu Feb  7 19:29:06 2002
@@ -41,7 +41,7 @@
 #	exit 1
 # fi
 
-config=/etc/ipsec.conf
+config=/etc/freeswan/ipsec.conf
 include=yes
 type=conn
 fieldfmt=yes
diff -Naur freeswan-1.95/utils/auto freeswan-1.95-p/utils/auto
--- freeswan-1.95/utils/auto	Thu Jan 31 21:28:59 2002
+++ freeswan-1.95-p/utils/auto	Thu Feb  7 19:29:19 2002
@@ -24,7 +24,7 @@
 	other options: [--config ipsecconfigfile] [--verbose] [--show]"
 
 showonly=
-config=/etc/ipsec.conf
+config=/etc/freeswan/ipsec.conf
 info=/var/run/ipsec.info
 shopts=
 noinclude=
diff -Naur freeswan-1.95/utils/barf freeswan-1.95-p/utils/barf
--- freeswan-1.95/utils/barf	Tue Nov 27 03:03:00 2001
+++ freeswan-1.95-p/utils/barf	Thu Feb  7 19:30:26 2002
@@ -144,15 +144,15 @@
 _________________________ ipsec/showdefaults
 ipsec showdefaults
 _________________________ ipsec/conf
-ipsec _include /etc/ipsec.conf | ipsec _keycensor
+ipsec _include /etc/freeswan/ipsec.conf | ipsec _keycensor
 _________________________ ipsec/secrets
-ipsec _include /etc/ipsec.secrets | ipsec _secretcensor
+ipsec _include /etc/freeswan/ipsec.secrets | ipsec _secretcensor
 _________________________ ipsec/ls-dir
-ls -l ${IPSEC_DIR-/usr/local/lib/ipsec}
+ls -l ${IPSEC_DIR-/usr/lib/ipsec}
 _________________________ ipsec/updowns
-for f in `ls ${IPSEC_DIR-/usr/local/lib/ipsec} | egrep updown`
+for f in `ls ${IPSEC_DIR-/usr/lib/ipsec} | egrep updown`
 do
-	cat ${IPSEC_DIR-/usr/local/lib/ipsec}/$f
+	cat ${IPSEC_DIR-/usr/lib/ipsec}/$f
 done
 _________________________ proc/net/dev
 cat /proc/net/dev
diff -Naur freeswan-1.95/utils/manual freeswan-1.95-p/utils/manual
--- freeswan-1.95/utils/manual	Fri Jun  1 17:49:13 2001
+++ freeswan-1.95-p/utils/manual	Thu Feb  7 19:30:46 2002
@@ -26,7 +26,7 @@
 unset LANG LANGUAGE LC_ALL LC_MESSAGES
 
 showonly=
-config=/etc/ipsec.conf
+config=/etc/freeswan/ipsec.conf
 info=/var/run/ipsec.info
 shopts=
 other=0
