--- freeswan-1.97/utils/Makefile	Mon Apr  1 22:05:27 2002
+++ freeswan-1.97/utils/Makefile.org	Wed May  1 18:41:09 2002
@@ -71,17 +71,19 @@
 
 # generate RSA private key... if, and only if, /etc/ipsec.secrets does
 # not already exist
-BLURB=-- not filled in because ipsec.secrets existed at build time --
+# BLURB=-- not filled in because ipsec.secrets existed at build time --
+#
+# This will be done in %post in spec file 
 secrets.eg:	secrets.proto ranbits rsasigkey
-	rm -f $@
-	umask 077 ; ( \
-		sed '/yyy/,$$d' secrets.proto ; \
-		if test ! -f $(CONFDIR)/ipsec.secrets ; \
-		then ./rsasigkey --verbose $(RSAKEYBITS) ; \
-		else echo "	# $(BLURB)" ; \
-		fi ; \
-		sed '1,/yyy/d' secrets.proto ; \
-	) | egrep -v RCSID >$@
+#	rm -f $@
+#	umask 077 ; ( \
+#		sed '/yyy/,$$d' secrets.proto ; \
+#		if test ! -f $(CONFDIR)/ipsec.secrets ; \
+#		then ./rsasigkey --verbose $(RSAKEYBITS) ; \
+#		else echo "	# $(BLURB)" ; \
+#		fi ; \
+#		sed '1,/yyy/d' secrets.proto ; \
+#	) | egrep -v RCSID >$@
 
 # force full secrets-file build
 newsecrets:
@@ -106,8 +108,8 @@
 		$(INSTMAN) $(PUB) $$f $(CMANDIR)/ipsec_$$f || exit 1 ; \
 		./manlink $(CMANDIR) ipsec_$$f ; \
 	done
-	test -f $(CONFDIR)/ipsec.secrets || \
-		$(INSTCONF) $(PRIV) secrets.eg $(CONFDIR)/ipsec.secrets
+#	test -f $(CONFDIR)/ipsec.secrets || \
+#		$(INSTCONF) $(PRIV) secrets.eg $(CONFDIR)/ipsec.secrets
 	test -f $(CONFDIR)/ipsec.conf || \
 		$(INSTCONF) $(PUB) conf.eg $(CONFDIR)/ipsec.conf
 	# main copy must go in RCDIR, BINDIR may not be mounted at boot time
